platform/android/export/export.cpp

```diff
@@ -1993,21 +2002,38 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 
 		// Validate the rest of the configuration.
 
-		String dk = p_preset->get("keystore/debug");
-
-		if (!FileAccess::exists(dk)) {
-			dk = EditorSettings::get_singleton()->get("export/android/debug_keystore");
-			if (!FileAccess::exists(dk)) {
+		String debug_keystore;
+		if (OS::get_singleton()->has_environment(ENV_KEYSTORE_PATH)) {
+			debug_keystore = OS::get_singleton()->get_environment(ENV_KEYSTORE_PATH);
+			if (!debug_keystore.is_empty() && !FileAccess::exists(debug_keystore)) {
```

Comment:
Seems like we should fallback to the editor-wide keystore path, or at least print a warning if `ENV_KEYSTORE_PATH` is present but empty.